app-misc/boxes/boxes-2.2.0.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
```

Comment:
Can you keep 1.1.2-r3 around for now?

We usually prefer to cleanup old in a separate commit, but in particular, because boxes has "stable keywords" (e.g. amd64 has no "~"), we have to wait and stable the new one in 30 days or so, then we can cleanup old.